src/hooks/queries/useGetMarketNFTFloorPrice.ts

```diff
@@ -0,0 +1,34 @@
+import { useQuery, UseQueryOptions } from '@tanstack/react-query';
+
+import { QueryKeysEnum } from '@/enums/queryKeys.enum';
+import { getMarketNFTFloorPrice } from '@/services/casperdash/market/market.service';
+
+type Params = {
+  tokenPackageHash?: string;
+  tokenId?: string;
+};
+
+export type UseGetMarketNFTOptions = UseQueryOptions<
+  number,
+  unknown,
+  number,
+  [string, string | undefined, string | undefined, string]
+>;
+
+export const useGetMarketNFTFloorPrice = (
```

Comment:
I didn't see usage of this hook anywhere ?

Reply:
Removed it